### PR TITLE
Docs (Example) : Add Redis Upstash session example

### PR DESCRIPTION
**Pull request by gavriguy**
An example of creating a custom Redis  session using `createFileSessionStorage`.

When deploying to serverless infrastructure, it's impossible to use the core `createFileSessionStorage` option as there is no persistent file storage.

In this example we use Redis (Upstash) to store the session info.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -1,32 +1,29 @@
-# Remix Examples
+# Redis Session Storage Using Upstash Example
```
Comment (MichaelDeBoey) [inline examples/README.md:1]: These changes should be reverted and entered in `examples/redis-upstash-session/README.md`

File: examples/redis-upstash-session/sessions/0159/ca19a17d3092 [hunk 2]
```diff
@@ -0,0 +1 @@
+{"data":{"myStoredData":"some data"},"expires":"2022-01-07T15:06:48.773Z"}
```
Comment (MichaelDeBoey) [inline examples/redis-upstash-session/sessions/0159/ca19a17d3092:1]: I think the `sessions` folder could be removed from the example, no? 🤔

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: examples/README.md (hunk 1 above)
Comment (gavriguy) [reply examples/README.md]: Sorry for that.

File: examples/redis-upstash-session/sessions/0159/ca19a17d3092 (hunk 2 above)
Comment (gavriguy) [reply examples/redis-upstash-session/sessions/0159/ca19a17d3092]: Yes it was a leftover. 10x

File: examples/README.md [hunk 3]
```diff
@@ -29,4 +29,4 @@ Most examples should:
 
 To create an example, simply copy/paste the [`template`](template) directory into a new one with a sensible name, make the changes you need for your example, update the `README.md` and open a PR!
 
-Thanks!
+Thanks!
```
Comment (kentcdodds) [inline examples/README.md:32]: 🧐

**Review by kentcdodds — APPROVED** [review]
Stellar. Thank you so much!

File: examples/README.md (hunk 3 above)
Comment (gavriguy) [reply examples/README.md:32]: @kentcdodds my reaction exactly :)

**Pull request merged**